### ISSUE TITLE
About 2d detection results

### ISSUE THREAD

**Issue by hellos0032** [knu-vis/L2D-Pose#2]
Thank you for your nice work. I have a small question.

This works learning 2d detector, so didn't you measure the 2d detection results using AP or PCKh metrics?(not training loss) 
I just wondering better 2d pose leads better 3d lifting results.

Thank you.

**Comment by hyeon0819** [comment]
Thank you for your great question.

You can find 2D pose detection results in the [supplementary material of our paper](https://openaccess.thecvf.com/content/ACCV2024/supplemental/Lee_Learning_2D_Human_ACCV_2024_supplemental.pdf). 
Our method slightly improves 2D pose detection while primarily targeting 3D pose estimation.

**Comment by hyeon0819** [comment]
Thank you for your great question.

You can find 2D pose detection results in the [supplementary material of our paper](https://openaccess.thecvf.com/content/ACCV2024/supplemental/Lee_Learning_2D_Human_ACCV_2024_supplemental.pdf). 
Our method slightly improves 2D pose detection while primarily targeting 3D pose estimation.

Better 2D pose doesn’t always guarantee better 3D pose, but our method, designed to improve 3D pose, also led to noticeable improvements in 2D pose as a result.

**Comment by hellos0032** [comment]
Thank you for your answer!

I have another small question.
Did you manually fine-tuned 2D detector model which was named FT-2D at the Table 2? 

Thank you

**Comment by hyeon0819** [comment]
Yes, in 'FT-2D' method, we fine-tuned the 2D pose detector using only 2D loss (original training method of HR-Net).
